src/settings/VideoSettings.tsx

```diff
@@ -22,6 +26,10 @@ export default function VideoSettings() {
     setConfig((prevConfig: any) => ({ ...prevConfig, [stateKey]: value }));
   };
 
+  if (!fpsOptions.includes(config.obsFPS)) {
```

Comment:
Why is this required? Shouldn't we be getting a default from electron store if it's not set?

Reply:
If someone edits their config by hand and types 120 or 144 or something, OBS will just use 60, but in order to show something meaningful in the dropdown. `FPSCommon` only supports a few common FPS values.